src/main/java/com/scottlogic/librarygradproject/BooksController.java

```diff
@@ -4,41 +4,34 @@
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
-
 @RestController
 @RequestMapping("/api")
 public class BooksController {
 
-    private BookRepository bookRepo;
+    private final BookService bookService;
 
     @Autowired
-    public BooksController(BookRepository bookRepository){
-        bookRepo = bookRepository;
+    public BooksController(BookService bookService){
+        this.bookService = bookService;
     }
 
     @RequestMapping(value = "/books", method = RequestMethod.GET)
-    public List<Book> getAll() {
-        return bookRepo.getAll();
-    }
+    public List<Book> getAll() {return bookService.findAll();}
 
     @RequestMapping(value = "/books/{id}", method = RequestMethod.GET)
-    public Book get(@PathVariable int id) {
-        return bookRepo.get(id);
-    }
+    public Book get(@PathVariable long id) { return bookService.findOne(id); }
```

Comment:
Have you manually changed all these to one line? I think it's not as clear

Reply:
We will have to agree on the approach, I just remember someone mentioning they prefer to have them in one line but could spread them over 3 lines if this is the preference.